pkg/kubectl/cmd/completion.go

```diff
@@ -103,6 +104,7 @@ func NewCmdCompletion(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 		ValidArgs: shells,
 	}
+	cmd.Flags().String("boiler-plate", "", "The name of the copyright header file, defaults to hardcoded header value.")
```

Comment:
Odd because this feels like the user can choose the licence himself (I know the actual use-case to export completions and check them in for origin; but kubectl completion is also a user-facing command).